src/component/TrashNote/TrashNote.jsx

```diff
@@ -10,6 +10,12 @@ const TrashNote = ({ deletedNotes }) => {
   const deleteNotefromTrash = (noteId) => {
     setDeletedNotes((prev) => prev.filter((note) => note._id !== noteId));
   };
+
+  const current = new Date();
```

Comment:
You can give the Date in the Note Context, after creating a new Note so that this step does not repeat in more than two components